spring-cloud-app-broker-autoconfigure/src/main/java/org/springframework/cloud/appbroker/autoconfigure/ServiceInstanceLogStreamAutoConfiguration.java

```diff
@@ -59,7 +59,7 @@ public WebSocketHandlerAdapter handlerAdapter() {
 	@Bean
 	public HandlerMapping logsHandlerMapping(StreamingLogWebSocketHandler webSocketHandler) {
 		Map<String, WebSocketHandler> map = new HashMap<>();
-		map.put("/logs/**/stream", webSocketHandler);
+		map.put("/logs/**", webSocketHandler);
```

Comment:
@spikymonkey this is the change that I don't like. Boot changed the default parser and it doesn't support wildcards in the middle of the template like that anymore. You can set a config option to reenable the ant style parser, but I'd rather not do that. All the tests pass, which I guess means it's still working as desired. 😅

Reply:
Looking again at this, I found this logging pattern in.  `StreamingLogWebSocketHandler`:

```
private static final UriTemplate LOGGING_URI_TEMPLATE = new UriTemplate("/logs/{serviceInstanceId}/stream");
```

Is that the only pattern we need to match? Assuming we don't need to match things like `/logs/multiple/path/segments/stream` then we don't need the Ant style wildcards.

Reply:
That's the only one for which I'm aware. 🤷‍♂️